style.css

```diff
@@ -187,3 +187,337 @@ body {
 .twitter_hov:hover {
   background-image: url("images/icons/twitter_disabled.svg");
 }
+
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  background-color: white;
+  padding: 120px 24px 0 24px;
+}
+
+.works-title {
+  display: grid;
+  justify-content: center;
+  height: 107px;
+  gap: 24px;
+}
+
+.works-title h2 {
+  color: #1c1a19;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 130%;
+  margin: 0;
+  height: 83px;
+}
+
+.works-content {
+  margin: 0;
+  padding: 0;
+}
+
+.works-content hr {
+  margin-bottom: 80px;
+  margin-top: 0;
+  border: 1px solid #212121;
+}
+
+.works-content img {
+  margin-bottom: 24px;
+  width: 100%;
+}
+
+.works-content h2 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 138%;
+  margin-top: 0;
+  margin-bottom: 12px;
+}
+
+.works-content p {
+  margin-top: 0;
+  margin-bottom: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  text-align: justify;
+  line-height: 150%;
+  color: #091e42;
+  display: grid;
+  order: 1;
+}
+
+.works-content ul {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+  margin-top: 0;
+  margin-bottom: 12px;
+  padding: 0;
+  gap: 12px;
+}
+
+.works-content li {
+  margin: 0;
+  border: 2px solid #8993a4;
+  padding: 10px;
+  color: #091e42;
+  font-size: 15px;
+  font-weight: 500;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+}
+
+.works-content button {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  background: #ff6b00;
+  color: #fff;
+  border: 1px #ff6b00;
+  padding: 12px;
+  width: 124px;
+  height: 48px;
+}
+
+.center-content {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  background-color: white;
+  padding: 24px 16px 0 16px;
+}
```

Comment:
- [ ] Please use `flexbox` instead of `grid` because you can only use the `grid` for all cards not for the content of the card and this is also [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) so kindly use `flexbox` :slightly_smiling_face: 

**Project Requirement**
- In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.